src/pages/en/guides/markdown-content.md

```diff
@@ -430,12 +430,12 @@ Astro applies the [GitHub-flavored Markdown](https://github.com/remarkjs/remark-
 ```js title="astro.config.mjs" ins={2,3,7,8,11}
 import { defineConfig } from 'astro/config';
 import remarkToc from 'remark-toc';
-import rehypeMinifyHtml from 'rehype-minify';
+import { rehypeAccessibleEmojis } from 'rehype-accessible-emojis';
```

Comment:
I ran some tests and Astro doesn’t actually support `rehype-preset-minify` either. The preset is an object like `{ settings, plugins }` which is apparently supported by `rehype.use(...)` but not when passed to Astro’s `rehypePlugins` array (maybe it should be supported, not sure?)

So I’ve just switched this out completely for something that does work and is cool: a plugin that wraps emojis in a span with an accessible label.

Reply:
Oooh, way cooler!  ❄️ 🚀  LGTM!